var/spack/repos/builtin/packages/py-pythonsollya/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPythonsollya(PythonPackage):
+    """Python wrapper for the Sollya library"""
+
+    homepage = "Python wrapper for the Sollya library"
+    url      = "https://gitlab.com/metalibm-dev/pythonsollya/-/archive/release-0.4.0-alpha0/pythonsollya-release-0.4.0-alpha0.tar.gz"
+
+    version('0.4.0-alpha0', sha256='faac899744c92b1d20980cadef309cd5610d79722322e97940ff142c207c41b5')
```

Comment:
I don't know if we want to default to an alpha release, as it likely isn't stable. It would be better to add `preferred=True` to 0.3.0